### ISSUE TITLE
FR: Hover over note markers to open corresponding page preview

### ISSUE THREAD

**Issue by HEmile** [esm7/obsidian-map-view#60]
With the new Obsidian Hover Editor plugin, the page preview when hovering over a link has become much more useful. This could be a very nice (and easy to implement!) feature for Map View.

For example, when holding cmd/ctrl down and hovering over a note for eg 200 ms, you can trigger the `link-hover` event like this:  
`this.plugin.app.workspace.trigger('link-hover', this.element, null, file.path, '')`

**Comment by esm7** [comment]
I tried to do this in the past but failed to programmatically scroll the preview to a specific location (which is necessary for inline locations). Have you seen it done somewhere?

**Comment by HEmile** [comment]
I happen to have implemented that 2 hours ago, with the help of NothingIsLost:

```javascript
                const line = edge.data().line;
                const passState = {
                  scroll: line,
                  line: line,
                  startLoc: {
                    line: line,
                    col: edge.data().start,
                    offset: 0,
                  } as Loc,
                  endLoc: {
                    line: line,
                    col: edge.data().end,
                    offset: 0,
                  },
                };
                this.plugin.app.workspace.trigger('link-hover', this.element, null, file.path, '', passState);
```

I haven't yet tested it without hover editor, though. But this seems to work well.

**Comment by esm7** [comment]
This seems to work really nicely! Released in 1.4.0 :sunglasses:
